### PR TITLE
addressing issue 56

### PR DESCRIPTION
**Pull request by greenhsu123**
addressing issue [56](https://github.com/Green-Software-Foundation/training/issues/56)

### CODE REVIEW

**Review by jawache — APPROVED** [review]

**Comment by greenhsu123** [comment]
looks good to me!

**Comment by jawache** [comment]
@greenhsu123 I re-wrote it a little to cover practices also, sending it back to you to counter review :)

File: docs/practitioner/introduction.md [hunk 1]
```diff
@@ -32,3 +32,17 @@ The following certification covers 6 key areas that a green software practitione
...
 
 Each of these chapters will introduce some new concepts and explain in detail why they are important in terms of the climate, and how you can apply them to your green software practices.
+
+## Principles, Patterns, and Practices.
+
+This course's key areas and content describe the **principles** of green software, a core set of competencies needed to define, run and build green software.
+
+A green software **pattern** is a specific example of how to apply one or more principles in a real-world example. Whereas principles describe the theory that underpins green software, patterns are the practical advice software practitioners can use in their software applications today. Patterns are vendor-neutral.
+
+Green software **practices** are patterns applied to a specific vendor's product and inform practitioners how to use that product in the most sustainable way possible.
```
Comment (greenhsu123) [inline docs/practitioner/introduction.md:42]: Should we also say "A green software pattern..." so it's consistent with line 40?

Comment (jawache) [reply docs/practitioner/introduction.md:42]: Makes sense will adjust!

**Pull request merged**